packages/appstore/src/constants/cfd-account-type-details.ts

```diff
@@ -0,0 +1,94 @@
+import { localize } from '@deriv/translations';
+
+export const mt5 = {
+    synthetic: {
+        title: localize('Deriv MT5 Synthetics'),
+        description: localize('Trade CFDs on our Synthetic Indices that simulate real-world market movement.'),
+        leverage: { key: () => localize('Leverage'), value: () => localize('Up to 1:1000') },
```

Comment:
can we rename `key` and `value` to `getKey` and `getValue` so that it's easy to understand that it's a function